microprofile/galleon-common/src/main/resources/modules/system/layers/base/io/smallrye/fault-tolerance/main/module.xml

```diff
@@ -36,6 +36,8 @@
     <dependencies>
         <module name="io.smallrye.config"/>
         <module name="io.smallrye.metrics"/>
+        <module name="io.smallrye.reactive.converters"/>
+        <module name="io.smallrye.common.annotation"/>
```

Comment:
Lets keep these alphabetical, so move this higher please.

Reply:
Fixed.